package.json

```diff
@@ -47,12 +46,16 @@
     "mocha-lcov-reporter": "^1.3.0",
     "mock": "^0.1.1",
     "nyc": "^15.1.0",
+    "request": "^2.88.0",
```

Comment:
I don't think we can move this library to be strictly a dev dependency. The `buildRequestor` function called in the top level `createClient` call pulls in the `create` method from the `httpRequestor` file. That `create` method does pull in the `request` library in the case there wasn't one provided in the `requestorCofig`. I don't think we can guarantee one will always be provided since it isn't strictly required to create the client.

There were talks to switch from `request` to either `axios` or `fetch`. We should restart those conversations and start making changes towards incorporating a different http request  library so we can remove the `request` dependency altogether.

Reply:
Spot on, my bad. Missed the lib usages.